modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -3,52 +3,73 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
```

Comment:
Has this file been added while I was editing?
If so, that's not goodI need to be commenting on text that is not changing while I am in it.
NOT EDITED